### DISCUSSION TITLE
Change zoom buttons for left-handed users

### DISCUSSION THREAD

**Discussion by bravewolfcz** [cgeo/cgeo#14489] [Ideas]
Possibility to change the position of the zoom in and zoom out buttons on the map to the left side for left-handed users. For better use of c:geo in one hand for left-handed users.

**Comment by fm-sys** [comment]
You can also double tab and hold for zooming in and out. For me that's enough and I don't use the zoom buttons although I'm right-handed.

Still I somehow understand the request.

**Reply by moving-bits** [reply]
Recent implementation of user-defined zoom handling (#15012) would allow easier implementation of this request for UnifiedMap.

**Comment by eddiemuc** [comment]
> You can also double tab and hold for zooming in and out.

I know double tap for zooming in, but not a one-handed gesture to zoom out. Is there one?

**Reply by ztNFny** [reply]
Double-Tab AND SWIPE (down for in, up for out). Works in most map-apps

**Reply by devemux86** [reply]
- **One finger zoom in**: double tap with one finger to zoom in.
- **Two fingers zoom out**: tap with two fingers to zoom out. (VTM)
- **One finger zoom in / out**: double tap with one finger, keep the finger on screen after second tap and move it up / down to zoom in / out.
